google-cloud-storage/lib/google/cloud/storage/file.rb

```diff
@@ -355,6 +355,25 @@ def content_type= content_type
           update_gapi! :content_type
         end
 
+        ##
+        # A custom time specified by the user for the file, or `nil`.
+        #
+        # @return [DateTime, nil]
+        #
+        def custom_time
+          @gapi.custom_time
+        end
+
+        ##
+        # Updates the custom time specified by the user for the file.
+        #
+        # @param [DateTime, nil] custom_time A custom time specified by the user for the file, or `nil`.
```

Comment:
Once set it can't be unset and only changed to a custom datetime in the future. If the custom_time must be unset, you must either perform a rewrite operation or upload the data again.

Reply:
Done.